docs/legacy/rke1_windows.md

```diff
@@ -0,0 +1,112 @@
+## RKE1 Windows
+
+> **Note**: RKE1 Windows is no longer supported. Please see https://www.suse.com/support/kb/doc/?id=000020684.
+>
+> This document only exists for posterity to identify and document how RKE1 Windows was designed and what methods were used to support it.
+
+### What is RKE1?
+
+RKE1 (Rancher Kubernetes Engine) is the legacy Kubernetes distribution that Rancher supported that was responsible for installing and upgrading the Kubernetes components that needed to exist on a server for it to be added to a cluster. It has since been superceded by RKE2 / k3s, which were designed around some of the pitfalls that were baked into RKE1's design.
```

Comment:
I would remove the language here around legacy or indicating that RKE1 has been replaced or superseded by RKE2. as this isn't true from a product management perspective. Anything that could lead a customer to believe that RKE1 is going away or being replaced by RKE2 should be avoided.

Reply:
good call-out. done.